src/App.tsx

```diff
@@ -1,10 +1,46 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ListofTodo } from './Components/ListofTodo/ListofTodo';
+import { Error } from './Components/Error/Error';
+import { FilterTodo } from './Components/FilterTodo/FilterTodo';
+import { FilterStatus } from './types/FilterStatus';
 
-const USER_ID = 0;
+const USER_ID = 6429;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filter, setFilter] = useState<FilterStatus>(FilterStatus.all);
+  const activeTodos = useMemo(() => {
```

Comment:
```suggestion
  const activeTodosСount = useMemo(() => {
```
better to rename it to `activeTodosСount` as this variable stores the array length